src/versions/2.0/json.ts

```diff
@@ -145,22 +136,92 @@ export const METADATA_DEFINITIONS = {
     },
     required: ["license_number", "state"],
   },
+  affirmation: {
+    type: "object",
+    properties: {
+      affirmation: {
+        const:
+          "To the best of its knowledge and belief, the hospital has included all applicable standard charge information in accordance with the requirements of 45 CFR 180.50, and the information encoded is true, accurate, and complete as of the date indicated.",
+      },
+      confirm_affirmation: {
+        type: "boolean",
+      },
```

Comment:
Does this need to specifically be `true`? Or is checking whether the value is `true` fall outside the scope of the validator?

Reply:
Though it is pretty minor, the actual evaluation of values (outside of ENUMs) should be left out of the validator. We'll see how blurry that line becomes...